test.sh

```diff
@@ -5,7 +5,7 @@ SECONDS=0
 
 set -e
 
-./validate.sh
+#./validate.sh
```

Comment:
any specific reason you're commenting out validate.sh?

Reply:
```
+ isort --check .
--
284 | ERROR: /codebuild/output/src769041754/src/github.com/awslabs/aws-data-wrangler/awswrangler/s3/_write.py Imports are incorrectly sorted and/or formatted.
285 | ERROR: /codebuild/output/src769041754/src/github.com/awslabs/aws-data-wrangler/awswrangler/s3/_read_parquet.py Imports are incorrectly sorted and/or formatted.
286 | ERROR: /codebuild/output/src769041754/src/github.com/awslabs/aws-data-wrangler/awswrangler/lakeformation/_read.py Imports are incorrectly sorted and/or
```

We have some `isort` problems in this branch. There was also some importance in import order in certain files which I think is contributing.